### ISSUE TITLE
Runner 'axe' returns invalid CSS selector starting with '>'

### ISSUE THREAD

**Issue by thomasfischer-his** [a11ywatch/kayle#19]
When running a11ywatch with `fast_axecore@4.6.31` and runner 'axe' to analyze https://www.vilhelmina.se, among other the following issue gets reported:

```json
    {
        "code": "meta-viewport-large",
        "type": "error",
        "typeCode": 1,
        "message": "Users should be able to zoom and scale the text up to 500%",
        "context": "<meta name=\"viewport\" content=\"width=device-width,  initial-scale=1.0, maximum-scale=2.0, user-scalable=yes\">",
        "selector": ">:nth-child(1)>:nth-child(3)",
        "runner": "axe",
        "recurrence": 0
      }
```

The selector starts with `>` which is AFAIK invalid. It may be the case that simply `html` is missing at the beginning, given that the context is about a `meta` tag, probably located inside `head`, but it should not be the user's task to guess here and correct invalid output.

**Comment by j-mendez** [comment]
@thomasfischer-his thank you for the issue. I agree, going to take a look today. The selector validation is a pain since there’s not a default lib that can handle the cases. I guess we are building that use-case through this :).

**Comment by j-mendez** [comment]
fixed `v0.7.13`
